data/cyp2d6_substrate_carbonmangels/meta.yaml

```diff
@@ -0,0 +1,77 @@
+name: cyp2d6_substrate_carbonmangels
+description: |-
+  CYP2D6 is primarily expressed in the liver. It is also highly expressed
+  in areas of the central nervous system, including the substantia nigra. TDC
+  used a dataset from Carbon Mangels et al, which merged information on substrates
+  and nonsubstrates from six publications.
+targets:
+- id: CYP2D6_Substrate
+  description: The drugs that are metabolized by the CYP P450 2D6(1) or not (0)
```

Comment:
how are the drugs identified here? SMILES? Pubchem ID? Drug net id?

Reply:
It has two identifier, SMILES and drug name or pubchem Synonymous. No pubchem id.

Reply:
It has two identifier, SMILES and drug name or pubchem Synonymous. No pubchem id.